financialaid/constants.py

```diff
@@ -58,7 +58,7 @@ class FinancialAidStatus:
     app_id=quote_plus(settings.OPEN_EXCHANGE_RATES_APP_ID)
 )
 
-FINANCIAL_AID_RESET_SUBJECT = "Financial aid review application is reset for {program_name} MicroMasters"
+FINANCIAL_AID_RESET_SUBJECT = "Your personalized course price for {program_name} MicroMasters"
```

Comment:
Sorry I didn't realize this is exactly the same as `FINANCIAL_AID_APPROVAL_SUBJECT` Let's differentiate it a bit:

`"Update to your personalized course price for {program_name} MicroMasters"`